Containerfile.buildah

```diff
@@ -1,38 +1,10 @@
-FROM registry.access.redhat.com/ubi9/ubi:latest@sha256:081c96d1b1c7cd1855722d01f1ca53360510443737b1eb33284c6c4c330e537c as builder
-RUN dnf -y install \
-    make \
-    golang \
-    glib2-devel \
-    gpgme-devel \
-    libassuan-devel \
-    libseccomp-devel \
-    git \
-    bzip2 \
-    runc \
-    containers-common
-
-WORKDIR /go/src/containers/buildah
-
-COPY buildah/ .
-
-RUN make buildah
-
-# # Rebase on ubi9
-# FROM registry.access.redhat.com/ubi9/ubi:latest@sha256:081c96d1b1c7cd1855722d01f1ca53360510443737b1eb33284c6c4c330e537c
-
-# COPY --from=builder /go/src/containers/buildah/bin/buildah /usr/bin/buildah
-
-# WORKDIR /workdir
-
-# RUN \
-#   groupadd -g 1000 buildah; \
-#   useradd -u 1000 -g buildah -s /bin/sh -d /home/buildah buildah
-
-# RUN chown -R buildah:buildah /workdir
-
-# USER buildah
-
-# ENTRYPOINT ["/usr/bin/buildah"]
-# Rebase on ubi9/buildah to insert our newer binary
-FROM registry.access.redhat.com/ubi9/buildah:latest
-COPY --from=builder /go/src/containers/buildah/bin/buildah /usr/bin/buildah
+# Source from buildah/contrib/docker/Dockerfile
+FROM Fedora
```

Comment:
- The capitalization here is strange
- I think we should prefer "normal form" images, in this case `quay.io/fedora/fedora:41` etc.

But to the larger point around needing a newer "unshare" command...there's also the option of using e.g. `quay.io/centos/centos:stream10` or so, which would make clear that we're just waiting for using the next RHEL version.